crates/libtiny_tui/src/tui.rs

```diff
@@ -1212,7 +1263,7 @@ impl TUI {
 
             MsgTarget::AllServTabs { serv } => {
                 for (tab_idx, tab) in self.tabs.iter().enumerate() {
-                    if tab.src.serv_name() == serv {
+                    if tab.src.serv_name().map_or(false, |s| s == serv) {
```

Comment:
If we were on nightly I could use `.contains()` here...

Reply:
You could do

```suggestion
                    if tab.src.serv_name() == Some(serv) {
```

Reply:
😮